plugin/doh/metrics.go

```diff
@@ -0,0 +1,30 @@
+package doh
+
+import (
+	"github.com/coredns/coredns/plugin"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+// Variables declared for monitoring.
+var (
+	RequestCount = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: plugin.Namespace,
+		Subsystem: "doh",
+		Name:      "requests_total",
+		Help:      "Counter of requests made per upstream.",
+	}, []string{"to"})
+	RcodeCount = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
This is the only one abbreviated to R. May be best to expand to RequestCodeCount

Reply:
On second viewing after further files it looks like this may be unnecessary as Rcode seems to have a specific meaning